features/main/debug/src/main/res/drawable/ic_baseline_music_note_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file should be located with the other drawables in the app. Or, at the very least, it should not be located in the `debug` feature, which has no relation to where we're using it